Production_scripts/Process_extracts/Produce_extract_tests/extract_maternity_tests.R

```diff
@@ -0,0 +1,43 @@
+####################################################
+# Name of file - produce-maternity-tests.R
+# Original Authors - Jennifer Thom
+# Original Date - March 2022
+# Written/run on - RStudio Server
+# Version of R - 3.6.1
+# Description - Produce tests for source linkage files:
+#               acute processed file.
+#####################################################
+
+# Load packages
+library(createslf)
+library(openxlsx)
+library(slfhelper)
+
+
+# Read in Data-----------------------------------------
+
+year <- "1920"
+
+# Read new data file
+new_data <- readr::read_rds(get_source_extract_path(year, "Maternity", ext = "rds"))
```

Comment:
issue with reading this in - should be fixed when functions updated (check_mode read/write)

Reply:
@Jennit07 / @catherineholland1 can we check this is fixed now, please? If Jen deletes the maternity rds file (which might already not exist) then re-create it with the latest functions/versions of the package. Hopefully, then everything will work for Catherine.

Reply:
I just re ran the maternity extract and saved out the .rds file and now it reads in error free